### PR TITLE
Update GraphQL data fetchers

### PR DESCRIPTION
**Pull request by ymorimo**
* Added new data fetchers for the `bulkPut`, `bulkDelete`, and `mutate` fields in the `Mutation` GraphQL type added in #424.
  * `DataFetcherHelper` has been introduced for the purpose of sharing common tasks among the data fetchers. The base class `DataFetcherBase` has been removed.
* A number of unit tests have been added and reorganized.

### CODE REVIEW

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you!

File: graphql/src/main/java/com/scalar/db/graphql/datafetcher/QueryScanDataFetcher.java [hunk 1]
```diff
@@ -14,38 +14,61 @@
...
   }
 
-  @SuppressWarnings("unchecked")
   @Override
   public Map<String, List<Map<String, Object>>> get(DataFetchingEnvironment environment)
       throws Exception {
     Map<String, Object> scanInput = environment.getArgument("scan");
+    Scan scan = createScan(scanInput);
+
+    // TODO: scan.withProjections()
```
Comment (feeblefakie) [inline graphql/src/main/java/com/scalar/db/graphql/datafetcher/QueryScanDataFetcher.java:39]: Is this forgotten? or just a TODO for upcoming PRs?

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
(Left one minor question though)

File: graphql/src/main/java/com/scalar/db/graphql/datafetcher/QueryScanDataFetcher.java (hunk 1 above)
Comment (ymorimo) [reply graphql/src/main/java/com/scalar/db/graphql/datafetcher/QueryScanDataFetcher.java:39]: Thank you! It’s going to be implemented in another PR.

**Pull request merged**